docs: improve English phrasing across upstream documentation (#2260)
tektoncd/pipelines-as-code: docs/content/docs/guide/authoringprs.md

  PipelineRun with all its Tasks as a single PipelineRun with no external
  dependencies.

- Inside your pipeline, you need to check out the commit as
  received from the webhook by checking out the repository from that ref. Most of the time
  you want to reuse the
  [git-clone](https://github.com/tektoncd-catalog/git-clone/tree/main/task/git-clone)
  task from the [tektoncd/catalog](https://github.com/tektoncd-catalog/git-clone/tree/main/task/git-clone).

- To be able to specify parameters of your commit and URL, Pipelines-as-Code
  gives you some “dynamic” variables that are defined according to the execution

>   of the events. These variables look like this `{{ var }}` and can be used

Reviewer: ```suggestion
  of the events. These variables are written as `{{ var }}` and can be used
```
